src/amo/withExperiment.js

```diff
@@ -26,6 +26,12 @@ import { getDisplayName } from 'amo/utils';
  *        in the experiment, assign a percentage to the special
  *        `NOT_IN_EXPERIMENT` variant.
  *
+ *  Note: The component will initially be rendered with `variant === null`.
+ *        Therefore the version of the component that will be the least
+ *        disruptive should be generated when `variant === null`. Once a
```

Comment:
We should probably explain that experiments are only being executed on the client before mentioning "disruptive versions".